tests/snappi_tests/multidut/systest/test_multiflow_pfcwd.py

```diff
@@ -0,0 +1,753 @@
+                if dut_list[0].hostname == port['peer_device']:
+                    logger.info('PFCWD stats for dut:{}, port:{},prio:{}'.
+                                format(dut_list[0].hostname, port['peer_port'], prio))
+                    pfcwd_stats = get_pfcwd_stats(dut_list[0], port['peer_port'], prio)
+                    logger.info('PFCWD Stats:{}'.format(pfcwd_stats))
+            else:
+                for dut in dut_list:
+                    if dut.hostname == port['peer_device']:
+                        logger.info('PFCWD stats for dut:{}, port:{},prio:{}'.
+                                    format(dut.hostname, port['peer_port'], prio))
+                        pfcwd_stats = get_pfcwd_stats(dut, port['peer_port'], prio)
+                        logger.info('PFCWD Stats::{}'.format(pfcwd_stats))
+
+    snappi_extra_params = SnappiSysTestParams()
+    snappi_extra_params.multi_dut_params.duthost1 = duthost1
+    snappi_extra_params.multi_dut_params.duthost2 = duthost2
+    snappi_extra_params.multi_dut_params.multi_dut_ports = snappi_ports
+
+    try:
+        run_pfc_test(api=snappi_api,
+                     testbed_config=testbed_config,
+                     port_config_list=port_config_list,
+                     conn_data=conn_graph_facts,
+                     fanout_data=fanout_graph_facts,
+                     global_pause=False,
+                     pause_prio_list=pause_prio_list,
+                     test_prio_list=test_prio_list,
+                     bg_prio_list=bg_prio_list,
+                     prio_dscp_map=prio_dscp_map,
+                     test_traffic_pause=True,
+                     test_def=test_def,
+                     snappi_extra_params=snappi_extra_params)
+
+        logger.info('PFC-WD stats at the end of the test:')
+        for prio in test_prio_list:
+            for port in snappi_ports:
+                if len(dut_list) == 1:
+                    if dut_list[0].hostname == port['peer_device']:
+                        pfcwd_stats = get_pfcwd_stats(dut_list[0], port['peer_port'], prio)
+                        logger.info('PFCWD Stats:for dut:{}, port:{},prio:{}, stats::{}'.
+                                    format(dut_list[0].hostname, port['peer_port'], prio, pfcwd_stats))
+                else:
+                    for dut in dut_list:
+                        if dut.hostname == port['peer_device']:
+                            pfcwd_stats = get_pfcwd_stats(dut, port['peer_port'], prio)
+                            logger.info('PFCWD Stats:for dut:{}, port:{},prio:{}, stats::{}'.
+                                        format(dut.hostname, port['peer_port'], prio, pfcwd_stats))
+
+    finally:
+        cleanup_config(dut_list, snappi_ports)
+
+
+@pytest.mark.parametrize('linecard_configuration_set', [config_set])
+@pytest.mark.parametrize('line_card_choice', line_card_choice)
+def test_pfcwd_frwd_90_10(snappi_api,                  # noqa: F811
```

Comment:
How to skip this case? Currently cisco devices don't support pfcwd forward mode.
Which part of script configures the pfcwd mode?

Reply:
I think the vendor-specific skip should be added by the corresponding team.